config/locales/ja.yml

```diff
@@ -1,15 +1,28 @@
 ---
 ja:
   about:
-    about_mastodon: Mastodon は<em>自由でオープンソース</em>なソーシャルネットワークです。商用プラットフォームの代替となる<em>分散型</em>を採用し、あなたのやりとりが一つの会社によって独占されるのを防ぎます。信頼できるインスタンスを選択してください &mdash; どのインスタンスを選んでも、誰とでもやりとりすることができます。 だれでも自分の Mastodon インスタンスを作ることができ、シームレスに<em>ソーシャルネットワーク</em>に参加できます。
+    about_mastodon_html: Mastodon は、オープンなウェブプロトコルを採用した、自由でオープンソースなソーシャルネットワークです。電子メールのように分散型の仕組みを採っています。
     about_this: このインスタンスについて
```

Comment:
🤔 it takes too much space

![](https://user-images.githubusercontent.com/1874549/28081775-5c953a42-66ab-11e7-82a8-253d5fccf9af.png)